kubernetes/samples/scripts/common/jrf-domain-template.yaml

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2017, 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Why does this look like a brand-new file?

Reply:
Previously same domain template was used for WLS and JRF samples for domain-home-in-image and domain-on-pv samples. Since we need to set different USER_MEM_ARGS (different heap sizes) for JRF domains, I created a new template which is used in JRF domain-home-in-image and domain-on-pv samples.